x-pack/plugins/session_view/public/types.ts

```diff
@@ -40,5 +39,4 @@ export interface DetailPanelProcessLeader {
   interactive: boolean;
   pid: number;
   entryMetaSourceIp: string;
-  executable: string;
```

Comment:
why no executable on ancestor processes? The ancestor processes should have all the same fields. except entry_leader which has the entry_meta stuff.

I just took a look at the figma and i can see why this was done. I would not take the field layout verbatim in that design, it's not correct. It looks like some sections were copy pasted too. The ECS examples are a better source of truth. But yea, the leader sections should have args, executable, pid etc.. start, end, etc... pretty much every prop in the top level process. The one difference with would be 'executable' as we would only show 1 value, and we wouldn't know if it was due to fork or exec. (since it's only a snapshot of that part of the lineage at the time of the event.